entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+if  [ -z "${IROHA_HOST}" ] || [ -z "${IROHA_PORT}" ]; then
+    echo "Please set IROHA_HOST and IROHA_PORT variables"
+    exit
+fi
+
+STATE=1
+
+while [ $STATE -ne 0 ]; do
```

Comment:
you do it by 'docker-compose-wait' no need to do it again manually

Reply:
This code is simply copied from D3 repo. I don't even know what the script does.